client/src/app/services/wordList.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable} from '@angular/core';
+import {Http} from '@angular/http';
+import {Word} from '../models/word';
+import {Observable} from "rxjs";
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class WorldListService {
+
+  constructor(private http: Http) {
+  }
+
+  getData():Observable<[Word]>{
+    return this.http.get('http://localhost:3000/api/listWord')
```

Comment:
Proponuje zrobić jeden plik który będzie miał link do api, np. 'config.ts' w folderze root, w którym będziesz miał obiekt z konfiguracją i linkiem do Api. I w tym pliku zaimportować tą stałą i się do niej odnieść. Wtedy, jeżeli zmienimy link do api, to zmienimy go w jednym miejscu. 
przykład: 
export API_URL = 'http://localhost:3000/api';

import {API_URL} from '../config.ts';

this.http.get(`${API_URL}/listWord`);